packages/babel-preset-react-app/index.js

```diff
@@ -27,7 +27,10 @@ const plugins = [
       regenerator: true,
       // Resolve the Babel runtime relative to the config.
       moduleName: path.dirname(require.resolve('babel-runtime/package'))
-    }]
+    }],
+    // const { a, ...z } = obj;
```

Comment:
Let's add a note here specifically mentioning why we need them, and link to the issue.